Firestore/core/test/firebase/firestore/testutil/testutil.cc

```diff
@@ -30,9 +30,12 @@ using model::DocumentComparator;
 using model::DocumentSet;
 using model::FieldMask;
 using model::FieldPath;
+using model::FieldTransform;
 using model::FieldValue;
 using model::ObjectValue;
 using model::Precondition;
+using model::TransformMutation;
```

Comment:
I'm not sure this using statement is helpful. We've got another collision between the testutil::$Name and model::$Name. Currently, all usages of model::TransformMutation are qualified, and unqualified usages refer to the testutil helper.

Reply:
Good call. Removed.